css/preview.css

```diff
@@ -0,0 +1,20 @@
+.node-preview-container {
+  padding: 0;
+  background-color: lightgrey;
+  position: relative;
+  box-shadow: -1px 0 3px 1px rgb(0 0 0 / 33%);
```

Comment:
It might be cool to have some shadows variables created too, so we can set them across the website for consistent shadows.